package.json

```diff
@@ -41,6 +36,11 @@
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "enzyme-to-json": "^3.4.4",
-    "gh-pages": "^2.2.0"
+    "gh-pages": "^2.2.0",
```

Comment:
`react-test-renderer` needs to be moved as well. It's not allowing me to comment on that line